modules/apps/roles/roles-test/src/testIntegration/java/com/liferay/roles/service/test/RoleServiceTest.java

```diff
@@ -84,6 +106,55 @@ public void testSearchCount() throws Exception {
 				new LinkedHashMap<>()));
 	}
 
+	@Test
+	public void testSearchWithClassNameId() throws Exception {
+		UserTestUtil.setUser(TestPropsValues.getUser());
+
+		_className = _classNameLocalService.addClassName(
+			RandomTestUtil.randomString());
+
+		_roles.add(
+			_roleService.addRole(
+				_className.getClassName(), RandomTestUtil.nextLong(),
```

Comment:
Is there any meaningful difference between `RandomTestUtil.nextLong()` here and `RandomTestUtil.randomLong()` in the subsequent calls?

Reply:
oops, good catch. I forgot to update the rest of the usages to `nextLong()`. nextLong calls the CounterLocalServiceUtil.increment, which ensures there's no duplicate of random long values.